### PR TITLE
Add more doc and link to setting up releases

### PR DESCRIPTION
**Pull request by mkangia**
Found myself a little lost trying to setup a private release on icds-staging the other day with specific branches.
So
1. Added an example in code itself for quick reference
2. Adding an example in the README for extensions
3. Added a link to README's relevant section in the common method called for both deploy and private releases, where i think it is relevant, though we might then need to rename the README title to "Manage releases" instead of "Private releases" if that makes sense?

##### ENVIRONMENTS AFFECTED

None

### CODE REVIEW

File: src/commcare_cloud/fab/README.md
Comment (snopoke) [inline src/commcare_cloud/fab/README.md:81]:
```suggestion
The same can be applied to set the branch for any additional repositories that have been configured. Use the variable `<name>_code_branch` where `name` is the name given to the repository in `meta.yml`:
```

File: src/commcare_cloud/fab/README.md
Comment (snopoke) [inline src/commcare_cloud/fab/README.md:84]:
```suggestion
cchq <env> fab setup_release --set <NAME>_code_branch=<BRANCH NAME>
```

**Review by snopoke — CHANGES_REQUESTED** [review]

**Review by snopoke — APPROVED** [review]
Thanks!

**Comment by mkangia** [comment]
Thanks @snopoke

**Comment by mkangia** [comment]
Hey @dannyroberts 
Since Simon and I co-contributed to this, going to wait for a review from you as well. Feel free to merge it directly if all looks okay.

**Review by dannyroberts — APPROVED** [review]

**Pull request merged**
